packages/server/src/integrations/postgres.ts

```diff
@@ -272,7 +272,8 @@ class PostgresIntegration extends Sql implements DatasourcePlus {
     try {
       const bindings = query.bindings || []
       this.log(query.sql, bindings)
-      return await client.query(query.sql, bindings)
+      const result = await client.query(query.sql, bindings)
+      return result
```

Comment:
NAB: Why?

Reply:
Ah, I did this so I could set a breakpoint with a way to check what was in here. Forgot to put it back when I was done.